src/url-loader/package-url-resolver.ts

```diff
@@ -30,55 +32,87 @@ export interface PackageUrlResolverOptions {
  * Resolves a URL to a canonical URL within a package.
  */
 export class PackageUrlResolver extends UrlResolver {
-  private packageUrl = `` as ResolvedUrl;
-  componentDir: string;
-  hostname: string|null;
+  private readonly packageDir: string;
+  private readonly packageUrl: ResolvedUrl;
+  readonly componentDir: string;
+  readonly hostname: string|null;
+  private readonly resolvedComponentDir: string;
 
   constructor(options?: PackageUrlResolverOptions) {
     super();
     options = options || {};
+    let packageDir = options.packageDir || process.cwd();
+    if (process.platform === 'win32') {
+      packageDir = packageDir.replace(/\\/g, '/');
```

Comment:
Does this handle escapes correctly? I'd rather delegate to a library to do this. What did we do for converting from Windows to URLs elsewhere?

Reply:
It does, windows doesn't use backslashes for escaping in paths.

I think it'd be good to use a library for handling windows URLs. Would you be ok with pushing windows comments to the next PR (https://github.com/Polymer/polymer-analyzer/pull/792) that adds windows testing?